### PR TITLE
DOCS-1739 Free tier subscription plan

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/user-guides/settings/subscriptions.md [hunk 1]
```diff
@@ -35,3 +35,14 @@ If you have recently registered in Wallarm Console and are using the [trial vers
 ![!Trial period](../../images/user-guides/settings/subscriptions/subscriptions-trial-with-poc.png)
 
 To get more information on the subscription to Wallarm API Security or to activate a paid subscription plan, please send a request to [sales@wallarm.com](mailto:sales@wallarm.com).
+
+## Free tier usage
+
+If you have registered in the US Cloud and are using the [Free Tier](../../about-wallarm-waf/subscription-plans.md#free-tier-subscription-plan) subscription plan, the subscription card displays the following information:
+
+* **Free Tier** subscription tile
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/subscriptions.md:43]: typo in tile

File: stylesheets/partials/header.html [hunk 2]
```diff
@@ -54,11 +54,11 @@
     
       <!-- Demo button -->
       <a href="https://us1.my.wallarm.com/signup?utm_source=wallarm_docs&utm_medium=button&utm_campaign=wallarm_docs_experiment_2" class="demo-button desktop-button">
-        Free Trial
+        Use for Free
```
Comment (AnastasiaTWW) [inline stylesheets/partials/header.html:57]: Maybe "Try for free"?

Comment (AnastasiaTWW) [reply stylesheets/partials/header.html:57]: because full usage for free is impossible

Comment (MaximBashurov) [reply stylesheets/partials/header.html:57]: I am not sure about it. Seems like "Try for free" in not much different from "Free Trial" - it is all abot "trial". But "Use for free" is about free usage which is closer to free usage unlimited in time (free tier).

May be we should invite somebody to this discussion? Third opinion would be benefitial.

Comment (AnastasiaTWW) [reply stylesheets/partials/header.html:57]: I suppose, Alexander will judge

**Pull request merged**